init/sql/migrate.sql

```diff
@@ -15,3 +15,9 @@ alter table package_version
 alter table package_version_latest
     alter column version_sort type varchar(30),
     alter column limited_version_sort type varchar(30);
+
+alter table org
+    add if not exists org_location varchar(30),
+    add if not exists org_env varchar(30),
+    add if not exists org_release varchar(30),
+    add if not exists org_status varchar(30);
```

Comment:
This should just be status,  but we already have a status.  So this needs to either use the existing status field, or use a more appropriate name that distinguishes it from our current status field, determined by what value is actually represents.